### PR TITLE
Redshift manually specifying which schema to parse 

### LINKED ISSUE #3: Redshift plugin: have an ability to explicitly tell the collector which schemas to parse

### PR DESCRIPTION
**Pull request by dudyshynyura**
Closes #3

### CODE REVIEW

File: odd_collector/domain/plugin.py [hunk 1]
```diff
@@ -39,6 +39,7 @@ class ClickhousePlugin(DatabasePlugin):
 
 class RedshiftPlugin(DatabasePlugin):
     type: Literal["redshift"]
+    schemas: list = []
```
Comment (Vixtir) [inline odd_collector/domain/plugin.py:42]: Use List from typing, with explicit type of elements

File: odd_collector/adapters/redshift/mappers/tables.py [hunk 2]
```diff
@@ -20,11 +20,13 @@ def map_table(
     mtables: MetadataTables,
     mcolumns: MetadataColumns,
     database: str,
+    schemas: List[str]
 ) -> List[DataEntity]:
     data_entities: List[DataEntity] = []
     column_index: int = 0
 
     for mtable in mtables.items:
+        if mtable.schema_name not in schemas and len(schemas): continue
```
Comment (Vixtir) [inline odd_collector/adapters/redshift/mappers/tables.py:29]: Odd collector has black formatter as dependency, use it with your IDE to format code

**Review by Vixtir — CHANGES_REQUESTED** [review]

File: odd_collector/domain/plugin.py (hunk 1 above)
Comment (dudyshynyura) [reply odd_collector/domain/plugin.py]: updated

File: odd_collector/adapters/redshift/adapter.py [hunk 3]
```diff
@@ -65,7 +68,7 @@ def get_data_entities(self) -> List[DataEntity]:
                 f"Load {len(mtables.items)} Datasets DataEntities from database"
             )
 
-            return map_table(self.__oddrn_generator, mtables, mcolumns, self.__database)
+            return map_table(self.__oddrn_generator, mtables, mcolumns, self.__database, self.__schemas)
```
Comment (DementevNikita) [inline odd_collector/adapters/redshift/adapter.py:71]: This way you still fetch everything and just map included schemas. It'd actually be better if you didn't fetch schemas you don't need to

File: odd_collector/adapters/redshift/adapter.py [hunk 4]
```diff
@@ -33,12 +33,15 @@ def __init__(self, config) -> None:
...
         self.__user = config.user
         self.__password = config.password
+        self.__schemas = config.schemas
 
         self._data_source = f"postgresql://{self.__user}:{self.__password}@{self.__host}:{self.__port}/{self.__database}?connect_timeout=10"
         self.__oddrn_generator = RedshiftGenerator(
             host_settings=f"{self.__host}", databases=self.__database
         )
 
+        logging.info(f"Redshift schemas to parse: {self.__schemas}")
```
Comment (DementevNikita) [inline odd_collector/adapters/redshift/adapter.py:43]: Please change it to debug

**Review by DementevNikita — CHANGES_REQUESTED** [review]

**Pull request closed without merging**